templates/tfengine/components/audit/main.tf

```diff
@@ -40,84 +40,67 @@ resource "google_{{.parent_type}}_iam_audit_config" "config" {
   }
 }
 
-# BigQuery log sink.
-resource "google_logging_{{.parent_type}}_sink" "bigquery_audit_logs_sink" {
-  name                 = "bigquery-audit-logs-sink"
-  {{$parent_field}}    = {{$parent_var}}
-  include_children     = true
-  filter               = "{{$filter}}"
-  destination          = "bigquery.googleapis.com/projects/${module.project.project_id}/datasets/${module.bigquery_destination.bigquery_dataset.dataset_id}"
+module "bigquery_export" {
+  source  = "terraform-google-modules/log-export/google"
+  version = "~> 5.0.0"
+
+  log_sink_name          = "bigquery-audit-logs-sink"
+  destination_uri        = "${module.bigquery_destination.destination_uri}"
+  filter                 = "{{$filter}}"
+  parent_resource_type   = "{{.parent_type}}"
+  parent_resource_id     = {{$parent_var}}
+  unique_writer_identity = true
+  include_children       = true
 }
 
 module "bigquery_destination" {
-  source  = "terraform-google-modules/bigquery/google"
-  version = "~> 4.3.0"
-
-  dataset_id                  = "{{.logs_bigquery_dataset.dataset_id}}"
-  project_id                  = module.project.project_id
-  location                    = "{{.bigquery_location}}"
-  default_table_expiration_ms = 365 * 8.64 * pow(10, 7) # 365 days
-  access = [
-    {
-      role          = "roles/bigquery.dataOwner",
-      special_group = "projectOwners"
-    },
-    {
-      role           = "roles/bigquery.dataViewer",
-      group_by_email = var.auditors_group
-    },
-  ]
+  source  = "terraform-google-modules/log-export/google//modules/bigquery"
+  version = "~> 5.0.0"
+
+  dataset_name             = "{{.logs_bigquery_dataset.dataset_id}}"
+  project_id               = module.project.project_id
+  location                 = "{{.bigquery_location}}"
+  log_sink_writer_identity = "${module.bigquery_export.writer_identity}"
+  expiration_days          = 365
 }
 
-resource "google_project_iam_member" "bigquery_sink_member" {
-  project = module.bigquery_destination.bigquery_dataset.project
-  role    = "roles/bigquery.dataEditor"
-  member  = google_logging_{{.parent_type}}_sink.bigquery_audit_logs_sink.writer_identity
-}
```

Comment:
do we need this or it's now part of the log-export module?

Reply:
This one's done by the cft module